manifests/run.pp

```diff
@@ -264,8 +264,8 @@
     $docker_command = $docker::params::docker_command
   }
 
-  $service_name = $docker::service_name
-  $docker_group = $docker::docker_group
+  $service_name = $docker::params::service_name
+  $docker_group = $docker::params::docker_group
```

Comment:
[The `docker` class inherits form `docker::params`](https://github.com/puppetlabs/puppetlabs-docker/blob/8f4b3d98c7110e3a4fc9151a14e55876eec2b585/manifests/init.pp#L476) and [allow personalization of these parameters](https://github.com/puppetlabs/puppetlabs-docker/blob/8f4b3d98c7110e3a4fc9151a14e55876eec2b585/manifests/init.pp#L439-L442) (using the values from `docker::params` for default values if not provided).

In order to access `$docker::*` variables, including this class should be enough:

```puppet
  include docker
```